src/dataprep/pipeline.sh

```diff
@@ -90,6 +97,17 @@ Rscript -e "rmarkdown::render('$script_path/reports/advisor_links_quality_select
 # ## Link NSF grants to MAG advisors
 bash $script_path/link/grants.sh $logfile_path
 
+
+# Manual split of biology!
+# bash $script_path/link/write_links_biology
```

Comment:
we should open an issue:
- tidy this up / make it automatic
- fix the jump in the links around the split year

Reply:
agree

Reply:
#19